modules/core/src/norm.simd.hpp

```diff
@@ -0,0 +1,711 @@
+        }
+        r0 = v_max(r0, v_max(r1, v_max(r2, r3)));
+        for (; j < n; j++) {
+            s = std::max(s, cv_abs(src[j]));
+        }
+        // [TODO]: use v_reduce_max when it supports float64
+        double t[VTraits<v_float64>::max_nlanes];
+        vx_store(t, r0);
+        for (int i = 0; i < VTraits<v_float64>::vlanes(); i++) {
+            s = std::max(s, cv_abs(t[i]));
+        }
+        return s;
+    }
+};
+
+template<>
+struct NormL1_SIMD<int, double> {
+    double operator() (const int* src, int n) const {
+        int j = 0;
+        double s = 0.f;
+        v_float64 r00 = vx_setzero_f64(), r01 = vx_setzero_f64();
+        v_float64 r10 = vx_setzero_f64(), r11 = vx_setzero_f64();
+        for (; j <= n - 2 * VTraits<v_int32>::vlanes(); j += 2 * VTraits<v_int32>::vlanes()) {
+            v_float32 v0 = v_abs(v_cvt_f32(vx_load(src + j))), v1 = v_abs(v_cvt_f32(vx_load(src + j + VTraits<v_int32>::vlanes())));
+            r00 = v_add(r00, v_cvt_f64(v0)); r01 = v_add(r01, v_cvt_f64_high(v0));
+            r10 = v_add(r10, v_cvt_f64(v1)); r11 = v_add(r11, v_cvt_f64_high(v1));
+        }
+        s += v_reduce_sum(v_add(v_add(v_add(r00, r01), r10), r11));
+        for (; j < n; j++) {
+            s += cv_abs(src[j]);
+        }
+        return s;
+    }
+};
+
+template<>
+struct NormL1_SIMD<float, double> {
+    double operator() (const float* src, int n) const {
+        int j = 0;
+        double s = 0.f;
+        v_float64 r00 = vx_setzero_f64(), r01 = vx_setzero_f64();
+        v_float64 r10 = vx_setzero_f64(), r11 = vx_setzero_f64();
+        v_float64 r20 = vx_setzero_f64(), r21 = vx_setzero_f64();
+        v_float64 r30 = vx_setzero_f64(), r31 = vx_setzero_f64();
+        for (; j <= n - 4 * VTraits<v_float32>::vlanes(); j += 4 * VTraits<v_float32>::vlanes()) {
+            v_float32 v0 = v_abs(vx_load(src + j)), v1 = v_abs(vx_load(src + j + VTraits<v_float32>::vlanes()));
+            r00 = v_add(r00, v_cvt_f64(v0)); r01 = v_add(r01, v_cvt_f64_high(v0));
+            r10 = v_add(r10, v_cvt_f64(v1)); r11 = v_add(r11, v_cvt_f64_high(v1));
+
+            v_float32 v2 = v_abs(vx_load(src + j + 2 * VTraits<v_float32>::vlanes())), v3 = v_abs(vx_load(src + j + 3 * VTraits<v_float32>::vlanes()));
+            r20 = v_add(r20, v_cvt_f64(v2)); r21 = v_add(r21, v_cvt_f64_high(v2));
+            r30 = v_add(r30, v_cvt_f64(v3)); r31 = v_add(r31, v_cvt_f64_high(v3));
+        }
+        s += v_reduce_sum(v_add(v_add(v_add(r00, r01), r10), r11));
+        s += v_reduce_sum(v_add(v_add(v_add(r20, r21), r30), r31));
+        for (; j < n; j++) {
+            s += cv_abs(src[j]);
+        }
+        return s;
+    }
+};
+
+template<>
+struct NormL1_SIMD<double, double> {
+    double operator() (const double* src, int n) const {
+        int j = 0;
+        double s = 0.f;
+#if CV_RVV // This is introduced to workaround the accuracy issue on ci
+        s = normL1_rvv(src, n, j);
+#else
```

Comment:
This is introduced to workaround failed tests on ci of rvv node.

The RVV CI node is getting weird some accuracy issues on the normL1 with double. 

But it works fine on real hardware. Could it be some qemu bugs?